operational/aws/long_running_instances/long_running_instances.pt

```diff
@@ -246,9 +253,9 @@ escalation "email" do
 end
 
 escalation "esc_stop_or_terminate" do
-  automatic false
+  automatic contains($param_automatic_action, "Stop/Terminate Instances")
```

Comment:
It looks like this policy as two actions, stop or terminate.  But the param_automatic_action only has one value.  There is also another `action` parameter that should be removed.

Reply:
Done!